<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntTestAppMainWindow</class>
 <widget class="QMainWindow" name="AntTestAppMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ANT+ Test Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="searchStickLabel">
      <property name="text">
       <string>Searching for ANT+ USB stick</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="InitializingLabel">
      <property name="text">
       <string>Intializing</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="heartRateLabel">
      <property name="text">
       <string>Heart Rate Sensor</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hrSearchLabel">
         <property name="text">
          <string>Searching..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hrSensorId">
         <property name="text">
          <string>Unknown ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentHrLabel">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Search Power Sensor</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Open Power Transmission Channel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>250</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="cadenceSpinBox">
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="hrMasterPushButton">
      <property name="text">
       <string>Open Heart Rate Master Channel</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="hrSpinBox">
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>130</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Search Smart Trainer</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Not Found</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="cadenceLabel">
      <property name="text">
       <string>0 rpm</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Cadence</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="powerLabel">
      <property name="text">
       <string>0 W</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Slope (in percent)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
